website/docs/5.0-upgrade-guide.html.markdown

```diff
@@ -148,6 +148,22 @@ Please follow the format in the example below for listing breaking changes in re
 * The deprecated `storage_account_name` property has been removed in favour of the `storage_account_id` property.
 * The deprecated `resource_manager_id` property has been removed in favour of the `id` property.
 
+### `azurerm_palo_alto_next_generation_firewall_virtual_hub_local_rulestack`
+
+* The `plan_id` property now defaults to `panw-cngfw-payg`.
+
+### `azurerm_palo_alto_next_generation_firewall_virtual_hub_panorama`
+
+* The `plan_id` property now defaults to `panw-cngfw-payg`.
+
+### `azurerm_palo_alto_next_generation_firewall_virtual_network_local_rulestack`
+
+* The `plan_id` property now defaults to `panw-cngfw-payg`.
+
+### `azurerm_palo_alto_next_generation_firewall_virtual_network_panorama`
+
+* The `plan_id` property now defaults to `panw-cngfw-payg`.
+
```

Comment:
The upgrade guide follows alphabetical ordering, these need to be moved up

Reply:
Updated.